src/woo_search/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -18,276 +18,331 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: woo_search/accounts/models.py:19
+#: src/woo_search/accounts/models.py:19
 msgid "username"
-msgstr ""
+msgstr "gebruikersnaam"
 
-#: woo_search/accounts/models.py:23
+#: src/woo_search/accounts/models.py:23
 msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
 msgstr ""
+"Vereist. 150 tekens of minder. Alleen letters, cijfers en @/,/+/-/_ tekens "
+"zijn toegestaan."
 
-#: woo_search/accounts/models.py:27
+#: src/woo_search/accounts/models.py:27
 msgid "A user with that username already exists."
-msgstr ""
+msgstr "Een gebruiker met deze gebruikersnaam bestaat al."
 
-#: woo_search/accounts/models.py:30
+#: src/woo_search/accounts/models.py:30
 msgid "first name"
 msgstr ""
 
-#: woo_search/accounts/models.py:31
+#: src/woo_search/accounts/models.py:31
 msgid "last name"
-msgstr ""
+msgstr "voornaam"
 
-#: woo_search/accounts/models.py:32
+#: src/woo_search/accounts/models.py:32
 msgid "email address"
-msgstr ""
+msgstr "achternaam"
 
-#: woo_search/accounts/models.py:34
+#: src/woo_search/accounts/models.py:34
 msgid "staff status"
-msgstr ""
+msgstr "stafstatus"
```

Comment:
You can leave this one empty, the translation shipped with Django itself will pick it up.